docs/api/hospital.md

```diff
@@ -168,4 +177,4 @@ response = requests.request("POST", url, headers=headers, data=payload)
 print(response.text)
 ```
 
-[EOF]
\ No newline at end of file
+[EOF]
```

Comment:
Este se lo puedes retirar, lo ideal es que vayamos quitandolos para mayor consitencia

Remove [EOF]

Reply:
El EOF no se quita - para formatos como md se utiliza para evitar truncar documentos. Si te refieres al newline, en general siempre dejamos un renglón libre al final del archivo precisamente para ser consistentes y porque hay [algunos formatos ](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) que lo necesitan. [Source below and some more deep sources screencapped]

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

<img width="764" alt="image" src="https://user-images.githubusercontent.com/8750702/162325877-8982c238-b985-405c-861c-c8261b2839db.png">